packages/components/src/vue/components/setting.vue

```diff
@@ -127,6 +135,12 @@ export default {
         return this.l10n.getText(this.dataModel.labelL10n)
       }
       return this.dataModel.labelText
+    },
+    labelHtml() {
+      if (this.dataModel.labelL10n) {
+        return this.l10n.getText(this.dataModel.labelL10n)
```

Comment:
I think we can get HTML from here can't we?

Reply:
yes!